FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -161,18 +161,48 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     because the text ends abruptly and one folio is missing in <locus target="q12"/>.
                                     Wooden boards broken and repaired.
                                     <locus target="#q16"/> has been repaired and attached to the codex with a more recent thread.
```

Comment:
`locus `is reserved for folia, to point to anything else we use `ref` with a particular `@type`
here `<ref type="quire" target="#q12"/>` `<ref type="quire" target="#q16"/>` etc

Reply:
Done.